core/fetcher.py

```diff
@@ -103,17 +104,26 @@ def fetch(self) -> list[Data]:
         return files
 
 
-class JSONFetcher(Fetcher):
-    def __init__(self, name: str, config: Config):
-        super().__init__(name, config)
-
-    def fetch(self):
-        pass
+# GZip generally compresses only one file, so file_path and file_name are not used
```

Comment:
`generally`; afaik, it is only ever a single "file" (really just any data stream). 's why it's combined with tar.